Source/WebCore/html/track/WebVTTTokenizer.cpp

```diff
@@ -209,7 +186,9 @@ bool WebVTTTokenizer::nextToken(WebVTTToken& token)
     }
 
 StartTagAnnotationState:
-    if (character == '>' || character == kEndOfFileMarker)
+    if (character == '&') {
+        WEBVTT_ADVANCE_TO(HTMLCharacterReferenceInAnnotationState);
+    } else if (character == '>' || character == kEndOfFileMarker)
```

Comment:
In WebKit coding style this would not have braces. I suggest omitting them.

Reply:
Done
https://github.com/WebKit/WebKit/compare/cec8ab3d2503e678a79df16e308353d9616529f1..256d17ff8be37d8154089884bf8faf30fecde728